flocker/node/agents/blockdevice.py

```diff
@@ -114,6 +114,21 @@ class DiscoveredDataset(PClass):
     )
 
 
+class DesiredDataset(PClass):
+    """
+    Dataset as requested by configuration (and applications).
```

Comment:
I don't really understand why "(and applications)" is in parentheses. Will they use `DesiredDataset` in a different way somehow? 

Could you please either remove the parens or expand on the difference in the docstring.